cw-orch-daemon/src/sync/builder.rs

```diff
@@ -23,9 +23,12 @@ use super::{super::error::DaemonError, core::Daemon};
 pub struct DaemonBuilder {
     // # Required
     pub(crate) chain: Option<ChainData>,
-    pub(crate) handle: Option<tokio::runtime::Handle>,
     // # Optional
+    pub(crate) handle: Option<tokio::runtime::Handle>,
     pub(crate) deployment_id: Option<String>,
+    pub(crate) additional_grpc_url: Option<String>,
```

Comment:
```suggestion
    pub(crate) overwrite_grpc_url: Option<String>,
```

Reply:
Should either rename this or update the URL logic to *add* the url to the grpc url list instead of replacing it alltogether.

Reply:
We need to be able to replace it in case the current one gives errors (even if we have fallback logic right now)
Choosing for replacing it (works in most cases)